lib/publicReportingAPI.js

```diff
@@ -1,9 +1,10 @@
 const ClientPublicReportingAPI = require('reportportal-client/lib/publicReportingAPI');
+const { RP_STATUSES } = require('reportportal-client/lib/constants/statuses');
 const LOG_LEVELS = require('./constants/logLevels');
 
 const PublicReportingAPI = {
   log: (level = LOG_LEVELS.INFO, message = '', file) =>
-    ClientPublicReportingAPI.addLog({ log: { level, file, message } }),
+    ClientPublicReportingAPI.addLog({ level, file, message }),
```

Comment:
After this change the tests of `publicReportingAPI.test.js` are failed. Could you please fix these.

Reply:
Fixed